views/signup.ejs

```diff
@@ -1,26 +1,4 @@
-<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/open-fonts@1.1.1/fonts/inter.min.css">
-<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@exampledev/new.css@1.1.2/new.min.css">
-
-
-<style>
-    * {
-        text-align: center;
-    }
-    input {
-        width: 400px;
-        height: 40px;
-        margin: 5px;
-    }
-    button {
-        width: 400px;
-        height: 40px;
-        margin: 10px;
-    }
-    .body {
-        margin: 10px;
-    }
-</style>
-
+<%- include('./partials/head') %>
```

Comment:
How to solve "EJS Unexpected token / - Using Partials" issue: https://cmsdk.com/node-js/ejs-unexpected-token---using-partials.html 👻